first_app/hello.py

```diff
@@ -109,56 +104,193 @@ def user_info_api(user_id):
     }, 200
     
     
-@hello_urls.route('/api/v1/user-posts/<user_id>', methods=['POST'])
+@hello_urls.route('/api/v1/create-post/<user_id>', methods=['POST'])
 @login_required
 def create_post(user_id):
-    posts = Post.query.all()
-    posts = []
-    for post in posts:
-        posts.append({
-        "id": post["id"],
-        "title": post["title"],
-        "body": post["body"],
-        "created": post["created"],	
-	})    
-    return json(posts)
-    
-
-@hello_urls.route('/api/v1/edit-posts/<user_id>', methods=['GET','POST'])
-def post_edit(user_id):
+    """Створення постів"""
     data = request.json
-    post = Post.query.get(user_id)
-    if request.method == "POST":
-        post.title = data['title']
-        post.body = data['body']
-        post.created = data['created']
-        db.session.commit()
-            		        
-    return "Post edited", 200	
+    title = data['title']
+    body = data['body']
+    created = data['create']	
+    author_id = None
+
+    post = Post(title, body, created, author_id)
+    db.session.add(post)
+    db.session.commit()
+
+    return {"Status": "Created post"}, 200
     
     
-@hello_urls.route('/api/v1/delete-post/<user_id>', methods=['GET','POST'])
-def delete_posts(post_id):
-    post = Post.query.filter_by(id=post_id, user_id=User.id).first()
+@hello_urls.route('/api/v1/delete-post/<user_id>', methods=['DELETE'])
+@login_required
+def delete_post(user_id):
+    """Видалення постів"""
+    post = Post.query.get(user_id)
```

Comment:
This is works? I think should be Post.query.get(Post.author_id == user_id)

Reply:
Yes, this is working. I checked code with Danila yesterday. Post deleted from DB.